.github/workflows/unit.yaml

```diff
@@ -0,0 +1,62 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+name: Unit Tests
+on:
+  push:
+
+env:
+  # uncomment this to enable an ngrok access to the vm on test failure
+  # enabling this you will get an ssh command to access the vm, active for one hour
+  #NGROK_DEBUG: yes
```

Comment:
Can we make this selectively enabled in the core repo by those who have permission?
IIRC, Travis does not enable the debug mode by default because secrets can be revealed.
(Though I am not sure there are any secrets with GHA too.)
If there are any, then we may need to keep them safe.

Reply:
Yes. I already made it optional and you can enable it triggering it explicitly.
You can run it on your fork.